README.md

```diff
@@ -266,32 +267,32 @@ type Rule =
     messageFilter : MessageFilter }
 ```
 
-And you can find the configuration on the module with the same name. The
-`Rule.empty` value is one where you accept any logs from anything.
+You can find the configuration in the module with the same name. The
+`Rule.empty` value is a null one that accepts all logs from anything.
```

Comment:
I always have a bit of a problem with all vs any. Wouldn't `any logs from anything` subsume `all logs from anything`, in a sense?